src/category_theory/limits/shapes/products.lean

```diff
@@ -6,6 +6,31 @@ Authors: Scott Morrison, Bhavik Mehta
 import category_theory.limits.has_limits
 import category_theory.discrete_category
 
+/-!
+# Categorical (co)products
+
+This file defines (co)products as special cases of (co)limits.
+
+A product is the categorical generalization of the object `Π i, f i` where `f : ι → C`. It is a
+limit cone over the diagram formed by `f`, implemented by converting `f` into a functor
+`discrete ι ⥤ C`.
+
+A coequalizer is the dual concept.
```

Comment:
```suggestion
A coproduct is the dual concept.
```
right?

Reply:
A lazy copy-paste on my part, you're completely correct, thanks!